src/utils/dateHelpers.ts

```diff
@@ -21,8 +21,11 @@ export function getISOString(potentialDate: string | undefined): string | undefi
 const locale = window.navigator?.language || (window.navigator as any)?.userLanguage || 'nb';
 moment.locale(locale);
 
-export function getDateFormat(format?: string): string {
-  return moment.localeData().longDateFormat('L') || format || DatepickerFormatDefault;
+export function getDateFormat(format?: string, selectedLanguage = 'nb'): string {
```

Comment:
I see you used a default value for selectedLanguage. However, to solve the issue in PDF, it also needs to pass this parameter in the other places this function gets used.

- You need to also include it in `Datepicker/index.ts` -> `useDisplayData()` for summary and PDF to work.
- `validateDatepickerFormData()` in `validation.ts` to get correct validation messages.
- Would also be nice to fix `InstanceInformationComponent` to get correct formatting for the date the instance was sent.
- In `ReceiptContainer`:123 the wrong formatting is also used.

Reply:
Thanks for feedback. 👍